### PR TITLE
Feature/ted 337

### PR DESCRIPTION
**Pull request by kaleanych**

### CODE REVIEW

File: ted_sws/notice_validator/resources/templates/sparql_query_results_report.jinja2 [hunk 1]
```diff
@@ -25,11 +53,11 @@
...
             <tr>
-                <th>{{ result.query.title }}</th>
-                <th>{{ result.query.description }}</th>
-                <th>{{ result.query.query }}</th>
-                <th>{{ result.result }}</th>
-                <th>{{ result.error }}</th>
+                <td>{{ result.query.title }}</td>
+                <td>{{ result.query.description }}</td>
+                <td>{{ result.query.query | replace('\n', '<br>') }}</td>
+                <td class="{% if result.result == "True" %}success{% else %}error{% endif %}">{{ result.result }}</td>
```
Comment (Dragos0000) [inline ted_sws/notice_validator/resources/templates/sparql_query_results_report.jinja2:59]: the result of a query might be something else than True Or False. We can have SELECT queries or ASK queries. You can leave only the {% if result.result == "True" %}success and delete the else statement

File: ted_sws/notice_validator/resources/templates/sparql_query_results_report.jinja2 [hunk 2]
```diff
@@ -25,11 +53,11 @@
...
+                <td>{{ result.error }}</td>
```
Comment (Dragos0000) [inline ted_sws/notice_validator/resources/templates/sparql_query_results_report.jinja2:60]: maybe put here the class error

File: ted_sws/notice_validator/services/sparql_test_suite_runner.py [hunk 3]
```diff
@@ -31,6 +34,11 @@ def __init__(self, rdf_manifestation: RDFManifestation, sparql_test_suite: SPARQ
         self.sparql_test_suite = sparql_test_suite
         self.mapping_suite = mapping_suite
 
+    @classmethod
+    def _sanitize_query(cls, query: str) -> str:
```
Comment (Dragos0000) [inline ted_sws/notice_validator/services/sparql_test_suite_runner.py:38]: why do we need this ?

Comment (Dragos0000) [reply ted_sws/notice_validator/services/sparql_test_suite_runner.py:38]: does it have a test ?

File: ted_sws/notice_validator/services/sparql_test_suite_runner.py [hunk 4]
```diff
@@ -94,6 +104,43 @@ def generate_html(self) -> RDFValidationManifestation:
...
+            created=datetime.now().isoformat(),
+            mapping_suite_identifier=mapping_suite_package.identifier,
+            query_result=query_result,
+            object_data="SPARQLQueryResultReport"
+        )
+
+        return sparql_result_report
+
+    @classmethod
+    def generate_result_html(cls, query_result: SPARQLQueryResult,
```
Comment (Dragos0000) [inline ted_sws/notice_validator/services/sparql_test_suite_runner.py:125]: why did we duplicated the methods ?

File: ted_sws/notice_validator/services/sparql_test_suite_runner.py [hunk 5]
```diff
@@ -94,6 +104,43 @@ def generate_html(self) -> RDFValidationManifestation:
 
         return RDFValidationManifestation(object_data=report)
 
+    @classmethod
+    def generate_result_json(cls, query_result: SPARQLQueryResult,
```
Comment (Dragos0000) [inline ted_sws/notice_validator/services/sparql_test_suite_runner.py:108]: why did we duplicated the methods ?

File: ted_sws/notice_validator/model/sparql_test_suite.py [hunk 6]
```diff
@@ -21,6 +21,16 @@ class SPARQLQueryResult(PropertyBaseModel):
     query: SPARQLQuery
     result: Optional[str]
     error: Optional[str]
+    identifier: Optional[str]
+
+
+class SPARQLQueryResultReport(RDFValidationManifestation):
```
Comment (Dragos0000) [inline ted_sws/notice_validator/model/sparql_test_suite.py:27]: not sure what this is for?

File: ted_sws/notice_validator/resources/templates/sparql_query_results_report.jinja2 (hunk 1 above)
Comment (kaleanych) [reply ted_sws/notice_validator/resources/templates/sparql_query_results_report.jinja2:59]: OK

**Review by costezki — APPROVED** [review]
looking fine

**Pull request merged**